packages/slash/src/withSlash.ts

```diff
@@ -0,0 +1,208 @@
+import { comboboxActions } from '@udecode/plate-combobox';
+import {
+  getEditorString,
+  getNodeString,
+  getPlugin,
+  getPointBefore,
+  getRange,
+  moveSelection,
+  PlateEditor,
+  setSelection,
+  TNode,
+  TText,
+  Value,
+  WithPlatePlugin,
+} from '@udecode/plate-common';
+import { Range } from 'slate';
+
+import { ELEMENT_SLASH_INPUT } from './createSlashPlugin';
+import {
+  findSlashInput,
+  isNodeSlashInput,
+  isSelectionInSlashInput,
+} from './queries/index';
+import { removeSlashInput } from './transforms';
+import { SlashPlugin, TSlashInputElement } from './types';
+
+export const withSlash = <
```

Comment:
At first, I wrote it inside the Mention package, but later I extracted it.Writing it together might make the code a bit messy and the package name will make users a bit confuse, while separating it might require maintaining two sets of code as well as test cases

Reply:
The common code will probably go to `combobox` package.